test/integration/orders.test.js

```diff
@@ -28,7 +28,8 @@ describe('Orders Integration', function () {
 
   it('supports placing orders in a `draft` state', async function () {
     const estimateResponse = await patch.estimates.createMassEstimate({
-      mass_g: 100
+      mass_g: 100,
+      create_order: true
```

Comment:
This was related to this change I had made: https://app.asana.com/0/1179991099298340/1200514022498599/f

This wasn't caught earlier by the health check because of this test being disabled.

Reply:
Nice catch!